_config.yml

```
@@ -4,7 +4,7 @@ exclude:             ['_includes/notes_graph.json']
 # You may need to change the base URL depending on your deploy configuration.
 # Specifically, when using GitHub Pages, the baseurl should point to where GitHub
 # Pages deploys your repository (which is usually the repository name).
-baseurl:             /
+baseurl:             
```

For clarity, we could use an empty string (rather than nothing at all):

```suggestion
baseurl:             ''
```

---

Ahhh! I didn't realize this was an option! Thanks for the suggestion!